Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -114,6 +114,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $purger->setPurgeMode($input->getOption('purge-with-truncate') ? ORMPurger::PURGE_MODE_TRUNCATE : ORMPurger::PURGE_MODE_DELETE);
         $executor = new ORMExecutor($em, $purger);
         $executor->setLogger(function ($message) use ($output) {
+            /*
+             * Hides the "Loading ... EmptyFixture" line, which is confusing.
+             * See the Fixture::getDependencies() method for details about this.
+             */
+            if (strpos($message, 'Doctrine\\Bundle\\FixturesBundle\\EmptyFixture') !== false) {
```

Comment:
Waiting for #209 would give us PHP 5.5, which would allow using the `::class` constant.